### ISSUE TITLE
Switch from Caffe pretrained model to pytorch pretrained model 

### ISSUE THREAD

**Issue by dljrichard** [jwyang/faster-rcnn.pytorch#356]
From Readme, 
> If you want to use pytorch pre-trained models, please remember to transpose images from BGR to RGB, and also use the same data transformer (minus mean and normalize) as used in pretrained model.

I wish to use pytorch model, and I referred to #10 solution, however, I still don't get satisfactory results. 
I think I may not use the transformer right. 

Could anyone indicates how to switch caffe pretrained model to pytorch model please?
Thanks sincerely in advance.

**Comment by kentaroy47** [comment]
#173 
Try this?
